test/internal/_metadata_finder_test.dart

```diff
@@ -35,6 +35,12 @@ void main() {
       }
     });
 
+    test('should get formats metadata for iso code going through references',
```

Comment:
Could you add a more high level test in phone_number_test for the expected format behavior, for example for a canadian phone number ?

Reply:
Absolutely! Incoming.